DotNetDBF/Utils.cs

```diff
@@ -36,93 +37,64 @@ static public byte[] FillArray(byte[] anArray, byte value)
             return anArray;
         }
 
-        static public byte[] trimLeftSpaces(byte[] arr)
+        public static byte[] trimLeftSpaces(byte[] array)
         {
-            List<byte> tList = new List<byte>(arr.Length);
+            List<byte> list = new List<byte>(array.Length);

```

Comment:
C# coding conventions always use implicit typing (aka `var`)  when the type of the variable is obvious.